commit b4d851b82da2181bfe744bd54ea6850652be241d (refs/changes/44/1444/2)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2017-02-06T22:07:15+00:00 (2 years, 8 months ago)
    
    TOOLS-1641 hermes-proxy cannot deal with lots of Manta load balancers

diff --git a/CHANGES.md b/CHANGES.md
index 62ea689..ecf34fb 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,10 @@
 # sdc (SDC ops core zone) Changelog
 
+## 1.4.0
+
+- Manta instances with a large number of load balancer IP addresses are
+  now supported by hermes, the log archival service (TOOLS-1641).
+
 ## 1.3.0
 
 - Add 'sdc-useradm *-attr' commands:
diff --git a/deps/hermes b/deps/hermes
index 417ef53..3702adf 160000
--- a/deps/hermes
+++ b/deps/hermes
@@ -1 +1 @@
-Subproject commit 417ef53987bc1ab264f470d6fb1c5d59761e9e89
+Subproject commit 3702adf367e3ea2a6b0c27889a0e9e0b4ac93ebc
diff --git a/package.json b/package.json
index 0b1b13b..c3f3881 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "sdc",
     "description": "SmartDataCenter 'sdc' zone to hold ops/admin tools",
-    "version": "1.3.0",
+    "version": "1.4.0",
     "author": "Joyent (joyent.com)",
     "private": true,
     "dependencies": {
